pkg/runtime/runtime_test.go

```diff
@@ -911,6 +913,7 @@ func TestReadInputBindings(t *testing.T) {
 		fakeReq := invokev1.NewInvokeMethodRequest("test")
 		fakeReq.WithHTTPExtension(http.MethodPost, "")
 		fakeReq.WithRawData([]byte("test"), "application/json")
+		fakeReq.WithMetadata(map[string][]string{})
```

Comment:
Can you please add one fake metadata to validate this new change ?

you need to add fake metadata in L925 ?
for example.
```
b := mockBinding{matadata: map[string]string{"bindings":"input"}
```

Reply:
@youngbupark Yeah, makes sense! 👍 I've updated the test with metadata.